recipes-ni/niconfigpersistentlogs/niconfigpersistentlogs_1.0.bb

```diff
@@ -0,0 +1,27 @@
+SUMMARY = "NI persistent logs configuration utility"
+DESCRIPTION = "Installs the niconfigpersistentlogs utility"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+SECTION = "base"
+
+DEPENDS += "shadow-native pseudo-native niacctbase update-rc.d-native"
```

Comment:
I don't think you need any of these `DEPENDS`.

* pseudo-native isn't required for any recent OE recipes, I don't think.
* shadow-native and niacctbase are only needed if you're using the labview user account or writing to its /home directory.
* update-rc.d-native will be included in the DEPENDS when you inherit `update-rc.d.bbclass`.

Reply:
Thanks for the explanation. I had meant to ask about this and forgot. I had just copied that line from similar recipes.